include/sd_sector_addresses.h

```diff
@@ -22,122 +22,251 @@ struct SectorAddress
 // clang-format off
 
 /**********
- * Start menu
+ * Start menu normaal
  **********/
-#define HOME_SIT_START SectorAddress { 0x0000, 0x0000 }
-#define HOME_SIT_START_SELECTED SectorAddress { 0x0000, 0x00A2 }
-#define HOME_SIT_START_ACTIVATED SectorAddress { 0x0000, 0x0051 }
+#define TURNED_ON SectorAddress { 0x0000, 0x2320 }
 
-#define STAND_UP SectorAddress { 0x0000, 0x1209 }
-#define STAND_UP_SELECTED SectorAddress { 0x0000, 0x12AB }
-#define STAND_UP_ACTIVATED SectorAddress { 0x0000, 0x125A }
+#define TURN_OFF SectorAddress { 0x0000, 0x2DE2 }
+#define TURN_OFF_SELECTED SectorAddress { 0x0000, 0x2E33 }
+#define TURN_OFF_ACTIVATED SectorAddress { 0x0000, 0x2E84 }
 
-#define HOME_STAND_START SectorAddress { 0x0000, 0x01E6 }
-#define HOME_STAND_START_SELECTED SectorAddress { 0x0000, 0x0288 }
-#define HOME_STAND_START_ACTIVATED SectorAddress { 0x0000, 0x0237 }
+#define HOME_SIT SectorAddress { 0x0000, 0x2371 }
+#define HOME_SIT_SELECTED SectorAddress { 0x0000, 0x23C2 }
+#define HOME_SIT_ACTIVATED SectorAddress { 0x0000, 0x2413 }
 
-#define TURN_OFF_START SectorAddress { 0x0000, 0x14E2 }
-#define TURN_OFF_START_SELECTED SectorAddress { 0x0000, 0x1584 }
-#define TURN_OFF_START_ACTIVATED SectorAddress { 0x0000, 0x1533 }
+#define STAND_UP SectorAddress { 0x0000, 0x2464 }
+#define STAND_UP_SELECTED SectorAddress { 0x0000, 0x24B5 }
+#define STAND_UP_ACTIVATED SectorAddress { 0x0000, 0x2506 }
+
+#define OBSTACLE_MENU SectorAddress { 0x0000, 0x2557 }
 /***** Start menu *****/
 
+#define WALK SectorAddress { 0x0000, 0x34D8 }
+#define WALK_SELECTED SectorAddress { 0x0000, 0x1075 }
+#define WALK_ACTIVATED SectorAddress { 0x0000, 0x10C6 }
+#define SINGLE_STEP_SELECTED SectorAddress { 0x0000, 0x1FA5 }
+#define SINGLE_STEP_ACTIVATED SectorAddress { 0x0000, 0x1FF6 }
+
 /**********
- * Walk menu
+ * Start menu 2
```

Comment:
Is there a more descriptive name than start menu 2? What is the difference with start menu 1?

Reply:
I believe this is the same menu but with different pictures? @juliagreeven?

Reply:
Does it make sense then to have 2?

Reply:
![Display13](https://user-images.githubusercontent.com/60779019/79433024-ea595380-7fcc-11ea-9b8c-489abda67a0d.JPG)
![IPDplaatjes_versie2113](https://user-images.githubusercontent.com/60779019/79433077-f80ed900-7fcc-11ea-9728-192b53f4648b.jpg)
Sorry inderdaad niet zo duidelijk, maar start menu 2 is nog een optie met de vierkantjes boven in, en normaal is dat zonder, met pijltjes links en rechts. En alleen start menu zijn de plain plaatjes zonder stippen en selected en activated plaatjes.

Reply:
Thanks for the clarification. I think the following will make it more clear when others want to change something in the future. 
```suggestion
 * Start menu squares at the top
```